packages/database/src/entity/Property.ts

```diff
@@ -0,0 +1,45 @@
+import { Column, Entity, ManyToOne, OneToMany } from 'typeorm';
+import { Document } from './Document';
+import { Field } from './Field';
+
+@Entity()
+export class Property {
+  @Column('text', { primary: true })
+  id?: string; // `${documentId}/${term}`
```

Comment:
This should be something other than text. It's currently stored in the form `${uuid}/${jsonTerm}`. Could use a UUID V5, but unsure how to query that. Will review when running performance and adding indexes.